pkg/kubelet/kubelet.go

```diff
@@ -512,6 +512,55 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 			if err != nil {
 				return nil, err
 			}
+		case "remote":
+			// Use the remote CRI shim for docker. This is needed for testing the
```

Comment:
I think the term "remote" is unclear in this context. Instead of reusing this term in the comment, you can clarify that by saying that "kubelet will talk to the shim over a unix socket using grpc. " and "This may become the default in the near future".